pkg/controller/transfer/user/setup.go

```diff
@@ -84,14 +83,6 @@ func postObserve(_ context.Context, cr *svcapitypes.User, obj *svcsdk.DescribeUs
 	return obs, nil
 }
 
-func postCreate(_ context.Context, cr *svcapitypes.User, obj *svcsdk.CreateUserOutput, cre managed.ExternalCreation, err error) (managed.ExternalCreation, error) {
-	if err != nil {
-		return managed.ExternalCreation{}, err
-	}
-	meta.SetExternalName(cr, awsclients.StringValue(obj.UserName))
-	return managed.ExternalCreation{}, nil
-}
-
 func preCreate(_ context.Context, cr *svcapitypes.User, obj *svcsdk.CreateUserInput) error {
 	obj.ServerId = cr.Spec.ForProvider.ServerID
 	obj.Role = cr.Spec.ForProvider.Role
```

Comment:
I wonder how that worked out before when nothing set the external name annotation